contracts/EVMCode.slb

```diff
@@ -0,0 +1,67 @@
+pragma solidity 0.5.2;
+pragma experimental ABIEncoderV2;
+
+
+library EVMCode {
+
+    struct Code {
+        address codeAddress;
+        bytes codeBytes;
+        uint length;
+    }
+
+    function fromAddress(address codeAddress) internal view returns (Code memory code) {
+        code.codeAddress = codeAddress;
+        uint codeSize;
+        assembly {
+            codeSize := extcodesize(codeAddress)
```

Comment:
why don't you assign `code.length` directly here :thinking:

Reply:
Good question; Going to fix that 👍

Reply:
Ha, I always forget this: That isn't possibly in an assembly block to (directly) assign a struct member (because memory)

Reply:
I think you can do 
`mstore(add(code, 0x60), extcodesize(codeAddress))`
but that's look not very nice :laughing:

Reply:
Yes, I wanted to do that at the second thought but didn't.
It is easy to forget about that and having strange errors if someone changes the structs.
We could make the offset a constant of course and putting this directly to the struct with a warning comment. Thoughts?

Reply:
solc needs a `offsetof` macro 😁

Reply:
let's skip this at the moment :)